docs/source/user-guide/concepts/packages.rst

```diff
@@ -1,13 +1,13 @@
-==============
-Conda packages
-==============
+========
+Packages
+========
 
 .. _concept-conda-package:
 
-What is a conda package?
-========================
+What is a package?
+==================
 
-A conda package is a compressed tarball file (.tar.bz2) or
+A package is a compressed tarball file (.tar.bz2) or
 .conda file that contains:
```

Comment:
```suggestion
A package is a compressed tarball file (``.tar.bz2``) or
``.conda`` file that contains:
```

Reply:
Actually, if we do this, we'd have to change all other instances (of which there are at lease several) on this page of ".conda" to "`.conda`" (fixed width), so maybe don't implement the suggestion above unless we're willing to make this change throughout the document

Reply:
I'm in favor of changing all of those to code snippets